torchvision/datasets/food101.py

```diff
@@ -11,7 +11,7 @@
 class Food101(VisionDataset):
     """`The Food-101 Data Set <https://data.vision.ee.ethz.ch/cvl/datasets_extra/food-101/>`_.
 
-    The Food-101 is a challenging data set of 101 food categories, with 101'000 images.
+    The Food-101 is a challenging data set of 101 food categories, with 101,000 images.
```

Comment:
The double comma might make it a bit hard to parse, might be worth dropping one of them

```suggestion
    The Food-101 is a challenging data set of 101 food categories with 101,000 images.
```